common_benchmark_suite/openxla/benchmark/comparative_suite/pt/scripts/generate_model_artifacts.py

```diff
@@ -84,10 +98,17 @@ def _parse_arguments() -> argparse.Namespace:
                       type=str,
                       default=".*",
                       help="The regex pattern to filter model names.")
+  parser.add_argument(
+      "--auto_upload",
```

Comment:
Follow the convention in other places, we can have `--auto-upload` and `--auto_upload`
```suggestion
      "--auto-upload",
      "--auto_upload",
```

Reply:
Added